packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.mjs

```diff
@@ -0,0 +1,348 @@
+/* eslint-disable @jessie.js/safe-await-separator */
+import { promises as fs } from 'fs';
+import assert from 'assert';
+
+import {
+  waitForBlock,
+  provisionSmartWallet,
+  getUser,
+  executeOffer,
+} from './upgradeHelpers.mjs';
+import {
+  agd,
+  agoric,
+  agops,
+  agopsLocation,
+  executeCommand,
+} from '../cliHelper.mjs';
+import {
+  HOME,
+  ATOM_DENOM,
+  GOV1ADDR,
+  GOV2ADDR,
+  GOV3ADDR,
+} from '../constants.mjs';
+
+const acceptEC = async () => {
+  console.log('ACTIONS Tickling the wallets so they are revived');
+
+  const govAccounts = [GOV1ADDR, GOV2ADDR, GOV3ADDR];
+  let voter = 0;
+  for (const account of govAccounts) {
+    const runs = [1, 2];
+    for (const run of runs) {
+      console.log(`${account}: ${run}: Accepting EC Committee`);
+
+      if (run === 1) {
+        try {
+          const params = ['ec', 'committee', '--send-from', account];
+          await executeCommand(agopsLocation, params, {
+            timeout: 30000,
+          });
+        } catch (error) {
+          console.warn(error.message);
+        }
+      } else {
+        try {
+          await agops.ec('committee', '--send-from', account, '--voter', voter);
+        } catch (error) {
+          console.warn(error.message);
+        }
+
+        voter += 1;
+      }
+
+      await waitForBlock(3);
+    }
+
+    console.log(`${account}: Accepting EC Charter`);
+    try {
+      await agops.ec('charter', '--send-from', account);
+    } catch (error) {
+      console.warn(error.message);
+    }
+  }
+};
+
+const acceptOracles = async oracles => {
+  for (const oracle of oracles) {
+    console.log(`${oracle.address}: Accept oracle invitations`);
+
+    const offerFunction = async () => {
+      return agops.oracle('accept', '--offerId', oracle.id);
+    };
+
+    await executeOffer(oracle.address, offerFunction);
+  }
+};
+
+const proposeNewAuctionParams = async (
+  address,
+  startFequency,
+  clockStep,
+  priceLockPeriod,
+) => {
+  const offerFunction = async () => {
+    const charterAcceptOfferId = await agops.ec(
+      'find-continuing-id',
+      '--for',
+      `${'charter\\ member\\ invitation'}`,
+      '--from',
+      address,
+    );
+
+    return agops.auctioneer(
+      'proposeParamChange',
+      '--charterAcceptOfferId',
+      charterAcceptOfferId,
+      '--start-frequency',
+      startFequency,
+      '--clock-step',
+      clockStep,
+      '--price-lock-period',
+      priceLockPeriod,
+    );
+  };
```

Comment:
This doesn't seem like an improvement over the shell syntax. Can we convert to JS style call? Would more work in the agops commands make this easier to support?

```
FASTER_AUCTIONS_OFFER=$(mktemp -t agops.XXX)
agops auctioneer proposeParamChange --charterAcceptOfferId "$(agops ec find-continuing-id --for "charter member invitation" --from "$GOV1ADDR")" --start-frequency $START_FREQUENCY --clock-step $CLOCK_STEP --price-lock-period $PRICE_LOCK_PERIOD >|"$FASTER_AUCTIONS_OFFER"
agoric wallet print --file "$FASTER_AUCTIONS_OFFER"
agops perf satisfaction --from "$GOV1ADDR" --executeOffer "$FASTER_AUCTIONS_OFFER" --keyring-backend=test
```